src/gui/app/templates/chat/_chat-messages.html

```diff
@@ -270,6 +270,7 @@ <h4><i class="fas" ng-class="currentThreadMessages().length < 2 ? 'fa-reply' : '
           style="font-size: 15px; display: flex; align-items: center"
         >
           <i
+            ng-if="activity.source.id + ':' + activity.event.id !== 'firebot:activity-feed-alert'"
```

Comment:
Given the changes we made to pass along event settings, we can remove the hardcoded check and do this instead:
```html
ng-if="activity.event.canRetrigger !== false"
```
(doing an explicit false check here just to ensure the default is showing the retrigger button)